server/app.py

```
@@ -96,79 +102,214 @@ class MergeBranchesRequest(BaseModel):
+
 class CreateConversationRequest(BaseModel):
     """Request model for creating a new conversation."""
     name: Optional[str] = None
 
-# Endpoints
+
+# --- Attachment Endpoints ---
+
+@app.post("/upload")
+async def upload_file(
+    file: UploadFile = File(...),
```

Ruff (0.14.13)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)